tests/introspection/tests.py

```diff
@@ -170,6 +170,9 @@ def test_get_primary_key_column(self):
     def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
+        if connection.features.can_introspect_index_type:
+            self.assertIn('type', indexes['reporter_id'])
```

Comment:
@timgraham Thanks for the review. I am aware of some improvements like this on this PR (which I realised when Jenkins started failing after I sent this PR). Will complete them by today and uncheck the "patch needs improvement" flag on the ticket.